package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "root",
+  "devDependencies": {
+    "lerna": "^6.4.1",
+    "node-fetch": "^2.6.7"
+  },
+  "scripts": {
+    "lint": "npx lerna exec -- npx eslint **/*.js"
```

Comment:
Commands here should not be aware of lerna.  Add eslint as dev dependency and change the command to `eslint **/*.js`

Reply:
And better to explicitly embed eslint config as `.eslntrc.js` or `.eslntrc.json`